docs/src/content/docs/getting-started/configuration.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Configuration 
```

Comment:
A "Basic Example" of a complete config.yml file at the end would help users see how the sections work together.